src/main/kotlin/com/vacgom/backend/infrastructure/auth/oauth/kakao/KakaoConnector.kt

```diff
@@ -0,0 +1,75 @@
+package com.vacgom.backend.infrastructure.auth.oauth.kakao
+
+import com.vacgom.backend.application.auth.dto.KakaoMemberResponse
+import com.vacgom.backend.application.auth.dto.OauthTokenResponse
+import com.vacgom.backend.domain.auth.oauth.OauthConnector
+import com.vacgom.backend.domain.auth.oauth.constants.ProviderType
+import com.vacgom.backend.global.exception.error.BusinessException
+import com.vacgom.backend.global.security.exception.AuthError
+import com.vacgom.backend.infrastructure.auth.oauth.kakao.model.KakaoProperties
+import org.springframework.http.HttpEntity
+import org.springframework.http.HttpHeaders
+import org.springframework.http.HttpMethod
+import org.springframework.http.MediaType
+import org.springframework.stereotype.Component
+import org.springframework.util.LinkedMultiValueMap
+import org.springframework.util.MultiValueMap
+import org.springframework.web.client.RestClientException
+import org.springframework.web.client.RestTemplate
+
+@Component
+class KakaoConnector(
+        private val restTemplate: RestTemplate,
+        private val kakaoProperties: KakaoProperties
+) : OauthConnector {
+    override fun isSupported(provider: ProviderType): Boolean {
+        return provider.isKakao()
+    }
+
+    override fun fetchOauthToken(code: String): OauthTokenResponse {
+        val headers = createHttpHeaders()
+        val body: MultiValueMap<String, String> = LinkedMultiValueMap()
```

Comment:
```suggestion
        val body = LinkedMultiValueMap<String, String>()
```

이런식으로 작성하시면 알아서 타입 추론을 통해 타입이 지정됩니다 !!